CHANGES.md

```diff
@@ -19,6 +19,15 @@
 * `xcube serve` now also serves datasets that are located in 
   subdirectories of filesystem-based data stores such as
   "file", "s3", "memory". (#579)
+
+* `xcube serve` no longer rejects datasets whose spatial
+  resolutions in x any differ less than 1%. (#590)
```

Comment:
As I see it, the datasets are still rejected, you have just changed the tolerance and added a message.

Reply:
Right. With rejection, tiles are misplaced, I already presented this today.

Reply:
> As I see it, the datasets are still rejected, you have just changed the tolerance and added a message.

Also refer to the name of this PR: _Revise spatial x,y resolutions check_

Reply:
Okay. What confused me was this part from the issue description:

> **Describe the solution you'd like**
> In such cases use datasets anyway and post warning.

which led me to believe that the solution you strived for was to also use datasets where the check has failed. The wording of the changelog entry (xcube serve no longer rejects datasets ...) did not help to make me realize I was wrong.
Suggestion for an alternative wording:

xcube serve now accepts datasets whose spatial resolutions differ up to 1%.